docs/how_to_contribute.md

```diff
@@ -15,85 +15,117 @@ And there are a lot of domains.
 
 # How to contribute
 
-Before contributing to Fundus you should check if you have fundus installed in `editable` mode and using dev-requirements.
+Before contributing to Fundus you should check if you have Fundus installed in `editable` mode and using dev-requirements.
 If you haven't done this so far or aren't sure about you should
 
 1. Clone the repository
-2. Optional but recommended: Create a virtualenv or conda environment
+2. Optional but recommended: Create a virtual environment (.venv) or conda environment
 3. Navigate to the root of the repository
 4. Run pip install -e .[dev]
 
 ## How to add a Publisher
 
-Before contributing a parser, check the [**readme**](../README.md) if there is already support for your desired publisher.
+Before contributing a parser, check the [**supported publishers**](supported_publishers.md) table if there is already support for your desired publisher.
 In the following, we will walk you through an example implementation of the [*Los Angeles Times*](https://www.latimes.com/) covering the best practices for adding a news source.
 
 ### 1. Creating a Parser Stub
 
 Take a look at the file structure in `fundus/publishers`.
-Fundus is divided into country-specific sections representing the country a news source originates from.
-For example
+Publishers covered are divided into country-specific sections representing the country a news source originates from.
```

Comment:
The beginning of this sentence could be improved. For example: "Fundus uses the codes from Alpha 2 to sort the publishers by country of origin. "

Reply:
Done